ISTC.js

```diff
@@ -0,0 +1,199 @@
+{
+	"translatorID": "29051e64-8eba-4b26-bbf1-0c224bc59497",
+	"label": "ISTC",
+	"creator": "Maike Kittelmann",
+	"target": "^https?://data\\.cerl\\.org/istc/(_search|i[a-z]\\d{8})",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcs",
+	"lastUpdated": "2016-09-28 13:09:43"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+	
+	RKE Web translator Copyright © 2016 Maike Kittelmann
+	This file is part of Zotero.
+	
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+	
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
+	
+	***** END LICENSE BLOCK *****
+*/
+
+
+function detectWeb(doc, url) {
+	if ( getSearchResults(doc, true, url) ) {
+		if (url.match('_search?')){
+			return "multiple";
+		}
+		else if (url.match(/i[a-z]\d{8}/)){
+			return 'book';
+		}
+	}
+}
+
+
+function getSearchResults(doc, checkOnly, url) {
+	var items = { };
+	var found = false;
+	if (url.match('_search?')){
+		var rows = ZU.xpath(doc, '//p[contains(@class, "ample-shortlist-item-entry")]/a[contains(@href, "/istc/i")]' );
+		for (i=0; i<rows.length; i++) {
+			var title = ZU.trimInternal(rows[i].textContent);
+			var href = rows[i].href + '?format=json';
+			if (!href || !title) continue;
+			if (checkOnly) return true;
+			found = true;
+			items[href] = title;
+		}
+	}
+	else {
+		var title = ZU.trimInternal( ZU.xpath(doc, '//div[contains(@class, "ample-record")]/h3' )[0].textContent );
+		if (!title) return false;
+		if (checkOnly) return true;
+		items[url] = title;
+	}

```

Comment:
Is the check important? I.e. are there cases such that the url check says 'yes' but it is no book described and we need an additional XPath test?